addons/stock/models/stock_picking.py

```diff
@@ -1126,6 +1128,32 @@ def action_detailed_operations(self):
             }
         }
 
+    def _get_next_transfers(self):
+        next_pickings = self.move_ids.move_dest_ids.picking_id
+        return next_pickings.filtered(lambda p: p not in self.return_ids)
+
+    @api.depends('move_ids.move_dest_ids')
+    def _compute_show_next_pickings(self):
```

Comment:
I know it's not perfect but we should try to strictly follow the python file guidelines 

https://www.odoo.com/documentation/master/contributing/development/coding_guidelines.html#symbols-and-conventions

compute methods should be together. Private/public ones too.